### PR TITLE
GDS 110 | use Jupyter password, not auth token

### PR DESCRIPTION
**Pull request by recursive-automata**
## Change Notes
* Added a script for generating necessary Jupyter notebook configuration object.
* Added a Make rule to create Jupyter notebook configuration file.
* Added a Make rule to start Jupyter in Docker and open a tab in Chrome 
* Modified Dockerfile to include the Jupyter configuration.

## Jira Issues
- [GDS-110](https://55places.atlassian.net/browse/GDS-110)

## Testing
Works on my machine. ¯\\\_(ツ)\_/¯

## Implementation
* Software Product

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/Makefile
Comment (davidzornek) [inline {{ cookiecutter.repo_name }}/Makefile:152]: Nice touch opening Chrome as part of the command. Couldn't possibly be more convenient.  With that, maybe even I will run my notebooks within the container!

**Review by davidzornek — APPROVED** [review]

File: {{ cookiecutter.repo_name }}/Makefile
Comment (jillrobinson) [inline {{ cookiecutter.repo_name }}/Makefile]: Could we please add a comment saying that you'll be requested to set a password here (for jupyter notebook)? And perhaps a note that you will need to remember that password for whenever you launch the container and jupyter?

Comment (jillrobinson) [reply {{ cookiecutter.repo_name }}/Makefile]: Could we make a JIRA ticket to update the README to reflect this process?

**Review by jillrobinson — APPROVED** [review]
Just a documentation request:

File: {{ cookiecutter.repo_name }}/Makefile
Comment (davidzornek) [reply {{ cookiecutter.repo_name }}/Makefile]: I had that thought as well, but for some reason didn't say it. Upon revisiting that thought, I incline toward saying that updating the readme is in scope for this PR.

Comment (davidzornek) [reply {{ cookiecutter.repo_name }}/Makefile]: Changing my approval to a change request to reflect my change of mind about the readme.

File: README.md
Comment (davidzornek) [inline README.md:70]: Approve with one change (you don't need to get another review before merging):
It's probably worth mentioning that simply typing `make` will list all available commands (but it excludes those rules associated with running the analysis; applies only to technical commands like this).

**Review by davidzornek — APPROVED** [review]

**Pull request merged**
